Template/Tag/MatomoTag.web.js

```diff
@@ -117,39 +128,167 @@
                     }
                     configuredTrackers[variableName] = tracker;
 
+                    var requireCookieConsentIndex = disableBrowserFeatureDetectionIndex
```

Comment:
Also can we document this, whenever any new config is supported we should not forget to add new variable here and support it.

Reply:
Sure. Do you think it would be best to note it in the developer documentation, the FAQ, or both? Any specific page recommendations?

Reply:
Developer documentation should be fine